velox/tool/trace/TraceReplayRunner.cpp

```diff
@@ -298,43 +299,49 @@ TraceReplayRunner::createReplayer() const {
         FLAGS_task_id,
         FLAGS_node_id,
         traceNodeName,
+        FLAGS_driver_ids,
         FLAGS_table_writer_output_dir);
   } else if (traceNodeName == "Aggregation") {
     replayer = std::make_unique<tool::trace::AggregationReplayer>(
         FLAGS_root_dir,
         FLAGS_query_id,
         FLAGS_task_id,
         FLAGS_node_id,
-        traceNodeName);
+        traceNodeName,
+        FLAGS_driver_ids);
   } else if (traceNodeName == "PartitionedOutput") {
     replayer = std::make_unique<tool::trace::PartitionedOutputReplayer>(
         FLAGS_root_dir,
         FLAGS_query_id,
         FLAGS_task_id,
         FLAGS_node_id,
         getVectorSerdeKind(),
-        traceNodeName);
+        traceNodeName,
+        FLAGS_driver_ids);
   } else if (traceNodeName == "TableScan") {
     replayer = std::make_unique<tool::trace::TableScanReplayer>(
         FLAGS_root_dir,
         FLAGS_query_id,
         FLAGS_task_id,
         FLAGS_node_id,
-        traceNodeName);
+        traceNodeName,
+        FLAGS_driver_ids);
   } else if (traceNodeName == "Filter" || traceNodeName == "Project") {
     replayer = std::make_unique<tool::trace::FilterProjectReplayer>(
         FLAGS_root_dir,
         FLAGS_query_id,
         FLAGS_task_id,
         FLAGS_node_id,
-        traceNodeName);
+        traceNodeName,
+        FLAGS_driver_ids);
```

Comment:
What if FLAGS_driver_ids is empty? Shall we extract all from the directory? thanks!